lib/element/command.js

```diff
@@ -394,15 +394,15 @@ class ElementCommand extends EventEmitter {
   }
 
   noSuchElementError(err) {
-    const {element, ms, abortOnFailure} = this;
+    const {element, ms, abortOnFailure, stackTrace:stack} = this;
```

Comment:
If this is going to be passed on every error, it's likely to flood your logs and make it harder to find the root cause. If this is necessary for some forms of debugging, I'd suggest either keeping the change local, or modifying the PR to wrap it in some kind of debug flag

Reply:
Also, I should add that you need to follow the checklist provided in the PR template.